scripts/lochness_create_template.py

```diff
@@ -377,6 +410,14 @@ def get_arguments():
                         action='store_true',
                         help='Enable lochness to lochness transfer on the '
                              'sender side')
+    parser.add_argument('-rsync', '--rsync',
+                        default=False,
```

Comment:
For future, `default=False` is not required for `action='store_true'`. It is **the** default.

Reply:
Thanks